### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by sachayves**
Add answers to the following questions:

- When do I need to top up my validator's balance?
- What's the difference between exiting and withdrawing?
- Can I stop running my validator for a few days and then start it back up again?

### CODE REVIEW

File: docs/the_nimbus_book/src/faq.md [hunk 1]
```diff
@@ -46,6 +46,26 @@ Limiting the maximum stake to 32 ETH encourages decentralization of power as it
...
 
+## Can I stop my validator for a few days and then start it back up again?
+
+Yes but, under normal conditions, you will lose an amount of ETH roughly equivalent to the amount of ETH you would have gained in that period. In other words, if you stood to earn ≈0.01 ETH, you would instead be penalised ≈0.01 ETH.
+
+## When should I top up my validator's balance?
+
+The answer to this question very much depends on how much ETH you have at your disposal.
+
+You should certainly top up if your balance is close to 16 ETH: this is to ensure you don't get kicked out of the validator set (which automatically happens if your balance falls below 16 ETH).
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/faq.md:57]:
```suggestion
You should certainly top up if your balance is close to 16 ETH: this is to ensure you don't get removed from the validator set (which automatically happens if your balance falls below 16 ETH).
```

File: docs/the_nimbus_book/src/faq.md [hunk 2]
```diff
@@ -46,6 +46,26 @@ Limiting the maximum stake to 32 ETH encourages decentralization of power as it
 
 > Remember that a validator’s vote is weighted by the amount it has at stake.
 
+## Can I stop my validator for a few days and then start it back up again?
+
+Yes but, under normal conditions, you will lose an amount of ETH roughly equivalent to the amount of ETH you would have gained in that period. In other words, if you stood to earn ≈0.01 ETH, you would instead be penalised ≈0.01 ETH.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/faq.md:51]: it's good to note here that the decay curve gets steeper as more time passes. there are some sites out there describing the economics of staying offline more thoroughly - might be good to consider showing a basic table?

**Review by arnetheduck — APPROVED** [review]

File: docs/the_nimbus_book/src/faq.md (hunk 2 above)
Comment (sachayves) [reply docs/the_nimbus_book/src/faq.md:51]: i think this is only true in the case where > 1/3 are offline (i.e. under abnormal conditions)?

Comment (sachayves) [reply docs/the_nimbus_book/src/faq.md:51]: as fas as i understand, if the chain is finalising, the penalties don't increase

**Pull request merged**